front/app/containers/IdeasEditPage/IdeasEditForm.tsx

```diff
@@ -314,63 +143,66 @@ const IdeasEditForm = ({ ideaId }: Props) => {
                     pb={isSmallerThanPhone ? '0' : '80px'}
                     overflowY="auto"
                   >
-                    {/* <InputSelectContext.Provider
-                      value={{
-                        onIdeaSelect: setSelectedIdeaId,
-                        title,
-                        body,
-                        setTitle,
-                        setBody,
-                        selectedIdeaId,
-                        showSimilarInputs,
-                      }}
-                    >
-                      <Form
-                        schema={schema}
-                        uiSchema={uiSchema}
-                        onSubmit={handleDisclaimer}
-                        initialFormData={initialFormData as FormValues}
-                        inputId={idea.data.id}
-                        getAjvErrorMessage={getAjvErrorMessage}
-                        getApiErrorMessage={getApiErrorMessage}
-                        config={'input'}
-                        loading={loading}
-                        showSubmitButton={false}
-                      />
-                    </InputSelectContext.Provider> */}
                     {project && (
                       <CustomFieldsForm
                         projectId={project.data.id}
                         phaseId={phaseId}
                         participationMethod={
                           phase?.data.attributes.participation_method
                         }
-                        initialFormData={idea.data.attributes}
+                        initialFormData={idea?.data.attributes}
                       />
                     )}
                   </Box>
                 </Box>
-                {/* <ContentUploadDisclaimer
-                  isDisclaimerOpened={isDisclaimerOpened}
-                  onAcceptDisclaimer={() => onAcceptDisclaimer(formData)}
-                  onCancelDisclaimer={onCancelDisclaimer}
-                /> */}
               </main>
             </Box>
-            {/* {selectedIdeaId && (
-              <Box
-                top={isSmallerThanPhone ? '0' : '40px'}
-                width="375px"
-                minWidth="375px"
-                borderLeft={`1px solid ${colors.grey300}`}
-                overflowY="auto"
-                bgColor={colors.white}
-                position="relative"
-                mb="80px"
-              >
-                <InputDetailView ideaId={selectedIdeaId} />
-              </Box>
-            )} */}
+            {selectedIdeaId &&
```

Comment:
Made this consistent between edit and new / potentially to be abstracted a bit later on.